content/docs/18-knowledgebase/00-how-to/01-how-to-setup-kube-api-audit-logging.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "How to Configure kube-apiserver audit logging"
+metaTitle: "Configuring the kube-apiserver audit logging featureo"
+metaDescription: "KnowledgeBase with How to"
+icon: "laptop"
+hideToC: true
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Getting Started with Palette and Auditing Your Kubernetes Cluster Using kube-apiserver Audit Logs
+
+This tutorial is based on the upstream Kubernetes documentation and kube-apiserver source code. It assumes that you have working knowledge of the Kubernetes API.
```

Comment:
The introduction should be written in an active voice and explain to the reader what they are about to learn. Something along these lines.

Kubernetes is a powerful container orchestration system that contains many features, including the ability to review cluster usage through audit logs. In this tutorial, you will learn how to audit your Kubernetes cluster by using the audit logs provided by the kube-apiserver.